modules/credential/src/test/java/org/eclipse/xpanse/modules/credential/CredentialCenterTest.java

```diff
@@ -258,7 +259,7 @@ void testGetCredentials_WithRequestedCredentialType() {
 
         credentialsStore.getCredential(csp, requestedCredentialType, xpanseUser);
 
-        assertEquals(Collections.singletonList(credential), result);
+        assertNotEquals(Collections.singletonList(credential), result);
```

Comment:
If the list is not same, we can now add assertions to validate if the values inside the list are same.

Reply:
Okay, I added it.